src/position.cpp

```diff
@@ -782,7 +810,7 @@ void Position::do_move(Move m, StateInfo& newSt, bool givesCheck) {
   assert(captured == NO_PIECE || color_of(captured) == (type_of(m) != CASTLING ? them : us));
   assert(type_of(captured) != KING);
 
-  // Remove gates. This might be too many gates when castling in Chess960!
+  // Remove gates. When castling 'to' will soon be modified so do this now.
```

Comment:
I have only one minor comment. I am not a native speaker, but at least for me this sentence is hard to read without commas. If setting the commas there is allowed, I would prefer
> When castling, 'to' will soon be modified, so do this now.

Reply:
Thanks for looking over this. I agree with your suggestion.